src/modules/commentCounter.test.js

```diff
@@ -0,0 +1,21 @@
+import { includeUI } from './displayComment.js';
+
+describe('includeUI', () => {
+  let mockCommentsContainer;
+  let mockCommentsNumber;
+
+  beforeEach(() => {
+    mockCommentsContainer = document.createElement('div');
+    mockCommentsNumber = document.createElement('span');
+    mockCommentsNumber.className = 'number';
+    mockCommentsContainer.appendChild(mockCommentsNumber);
+    document.querySelector = jest.fn(() => mockCommentsContainer);
+  });
+
+  it('should increment the comment counter', () => {
+    const initialCounter = mockCommentsNumber.textContent;
+    includeUI('testUser', 'testComment');
+    const updatedCounter = mockCommentsNumber.textContent;
+    expect(parseInt(updatedCounter, 10)).toBe(parseInt(initialCounter, 10) + 1);
+  });
+});
```

Comment:
- [ ] Kindly refactor your test file for testing the comment counter. Currently, it is not importing the function that calculates the number of comments of a given movie/series, and the mock you created is returning undefined objects when the tests run. You could create a mock HTML as you created in the other test file and import the function that counts the number of comments and test if the number is correct. Please, remember to test edge cases if there are no comments to count.
 - 
<img width="676" alt="image" src="https://user-images.githubusercontent.com/61919617/225652643-9aa75ad4-cd2d-4df5-83eb-0e5689c7ddee.png">